src/EdFi.AnalyticsMiddleTier.DataStandard33/Chrab/PostgreSQL/0002-View-ChronicAbsenteeismAttendanceFact-Create.sql

```diff
@@ -3,9 +3,9 @@
 -- The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
 -- See the LICENSE and NOTICES files in the project root for more information.
 
-DROP VIEW IF EXISTS analytics.chrab_ChronicAbsenteeismAttendanceFact;
+--DROP VIEW IF EXISTS analytics.chrab_ChronicAbsenteeismAttendanceFact;
 
-CREATE OR REPLACE VIEW analytics.chrab_ChronicAbsenteeismAttendanceFact
+CREATE OR ALTER VIEW analytics.chrab_ChronicAbsenteeismAttendanceFact
```

Comment:
The problem with the create or replace is that when the number of columns change, or the column types change Postgres doesn't like this function. That is why we do a drop if exists first, to then create the view again.
create or alter doesn't exist in Postgres.